src/codegeneration/ModuleTransformer.js

```diff
@@ -276,8 +276,8 @@ export class ModuleTransformer extends TempVarTransformer {
     assert(this.moduleName);
     let name = tree.token.processedValue;
     // import/module {x} from './name.js' is relative to the current file.
-    let normalizedName = System.normalize(name, this.moduleName);
-    return parseExpression `System.get(${normalizedName})`;
+    let normalizedName = $traceurRuntime.normalizeModuleName(name, this.moduleName);
```

Comment:
This one is more fundamental, it's the connection between compiled code and the module loader. Here I think we should put the normalization call in the generated code.  

But what generated code? If we had a standard then one implementation in the compiler would work for all loaders.  The compiler would be independent of the Loader; each loader would implement one or two functions mapping names to modules.  

But our current implementation essentially inlines the module-format-specific module-lookup code.  I guess we'll stick with that for now and emit `$traceurRuntime` calls matching our loader.  

@guybedford I guess uses of systemjs always compile with `--modules=instantiate` ?